impl/src/util.rs

```diff
@@ -436,48 +400,4 @@ mod tests {
       get_builtin_target_by_triple(triple).unwrap();
     }
   }
-
-  #[test]
-  fn generate_condition_strings() {
```

Comment:
Why are these tests not necessary?

Reply:
The functions for doing targets were broken out into three steps that essentially needed each other to actually work.

I made it one function since the API was scattered, in the process the tests that worked on the intermetidate steps are harder (and probably dont actually need testing as a unit) to test.

As I see it the function should go to being `fn(cargo_target) -> bazel_target`, rather than juggling through several functions. I can beef up the tests for targeting if wanted to cover the testing gap.